dist/rpm/openQA.spec

```diff
@@ -345,11 +349,7 @@ mkdir %{buildroot}%{_localstatedir}/lib/openqa/webui/cache
 #
 %fdupes %{buildroot}/%{_prefix}
 
-%pre
-if ! getent passwd geekotest > /dev/null; then
-    %{_sbindir}/useradd -r -g nogroup -c "openQA user" \
-        -d %{_localstatedir}/lib/openqa geekotest 2>/dev/null || :
-fi
+%pre -f %{name}.pre
```

Comment:
and this file is where?

Reply:
it is generated by `%sysusers_generate_pre`